dozer-spring-boot-auto-configuration/dozer-spring-boot-autoconfigure/src/main/java/com/github/dozermapper/autoconfigure/DozerConfigurationProperties.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
am wondering if this class should be moved to dozer-spring or dozer-core, and then the springboot class can extend and add any annotations needed.

Reply:
It can be like that, but what for?